velox/exec/DistinctAggregations.cpp

```diff
@@ -103,6 +103,20 @@ class TypedDistinctAggregations : public DistinctAggregations {
     inputForAccumulator_.reset();
   }
 
+  void addSingleGroupSpillInput(
+      char* group,
+      const VectorPtr& input,
+      vector_size_t index) override {
+    auto* arrayVector = input->as<ArrayVector>();
+    auto* elementsVector = arrayVector->elements()->asFlatVector<StringView>();
+
+    const auto size = arrayVector->sizeAt(index);
+    const auto offset = arrayVector->offsetAt(index);
+    auto* accumulator = reinterpret_cast<AccumulatorType*>(group + offset_);
+
+    accumulator->addFromSpill(*elementsVector, allocator_);
```

Comment:
It is a bit strange that size and offset are not used. Is there a bug?

Reply:
We are writing all the serialized data in a single entry